modules/guild.js

```diff
@@ -33,6 +37,7 @@ const fetchOrCreate = async (ctx, user, discord_guild) => {
         guild.botchannels = [ctx.msg.channel.id]
         guild.reportchannel = ctx.msg.channel.id
         guild.nextcheck = asdate.add(new Date(), 20, 'hours')
+        guild.prefix = "->"
```

Comment:
If this is a default value, maybe better to put it into the model

Reply:
I'll change this

Reply:
Resolved in latest commit